src/Admin.php

```diff
@@ -98,7 +98,7 @@ public static function updateSalePercentage($check, $object_id, $meta_key, $meta
         $parent_id = $product->get_parent_id();
         static::saveSalePercentage($parent_id, get_post($parent_id));
       }
-      elseif ($product->get_type() === 'simple') {
+      else {
```

Comment:
As we discussed in person, I'm a bit scared of removing the condition. There might be more edge cases. 

What if a third party plugin is adding another product type like bundle did? Would be the following statement correct in that case? 

Would not be safer to add an logical or on the condition to include bundles?
Like `elseif ($product->get_type() === 'simple' || $product->get_type() === 'bundle')`.